app/main.py

```diff
@@ -1,34 +1,65 @@
+from typing import Union
+
+
 class Deck:
-    def __init__(self, row, column, is_alive=True):
-        pass
+    def __init__(self, row: int, column: int, is_alive: bool = True) -> None:
+        self.row = row
+        self.column = column
+        self.is_alive = is_alive
 
 
 class Ship:
-    def __init__(self, start, end, is_drowned=False):
-        # Create decks and save them to a list `self.decks`
-        pass
+    def __init__(
+            self,
+            start: tuple,
+            end: tuple,
+            is_drowned: bool = False
+    ) -> None:
+        self.is_drowned = is_drowned
+        self.decks = []
+        if start == end:
+            self.decks.append(Deck(start[0], start[1]))
+        elif start[0] == end[0]:
+            current_column = start[1]
+            while current_column != end[1] + 1:
+                self.decks.append(Deck(start[0], current_column))
+                current_column += 1
+        elif start[1] == end[1]:
+            current_row = start[0]
+            while current_row != end[0] + 1:
+                self.decks.append(Deck(current_row, start[1]))
+                current_row += 1
```

Comment:
So hard code. Use loop inside loop instead of checks

Reply:
can't imagine it... i will try to use just two loops